### ISSUE TITLE
-install- all existing posts uneditable, not able to create new posts - This block has encountered an error and cannot be previewed

### ISSUE THREAD

**Issue by orionsweb** [WordPress/gutenberg#8690]
**Describe the bug**
Hello, 
Installed gutenberg, unable to edit any existing posts nor create new posts. 
Title blocks are alright in either instance however cannot edit nor create content, clicking on the first black block produces: This block has encountered an error and cannot be previewed. error message.

**Screenshots**
![image](https://user-images.githubusercontent.com/11905904/43802306-61a5a192-9a63-11e8-9855-4d8c3b881c91.png)

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser Chrome
 - Version 68.0.3440.84 (Official Build) (64-bit)

**Additional context**
Using Genesis w FocusPro child theme
Everything is up to date
Disabled all plugins still having issue

**Comment by youknowriad** [comment]
You probably have some javascript errors on the console. These would be handy to debug this issue. It looks like a theme/plugin incompatibility issue at first sight.

**Comment by orionsweb** [comment]
Turned off all plugins and even went to default theme nada.  

I've also reinstalled core and reinstalled Gutenberg.

Server setting?

**Comment by youknowriad** [comment]
If you spot the JavaScript errors in the console or the network failures (XHR requests), this might help? Are you using Cloudflare by any chance?

**Comment by Aliyanna** [comment]
I have exact the same problem

**Comment by DionDesigns** [comment]
Here are the JS errors that occur when loading a post for editing with the 3.4 version of Gutenberg:

`ReferenceError: tinymce is not defined  tinymce-plugin-lists.min.137fa94a.js:1:30
ReferenceError: tinymce is not defined  tinymce-plugin-paste.min.f6e82d7a.js:1:143
ReferenceError: tinymce is not defined  `tinymce-plugin-table.min.3fa6c17e.js:1:42``

This error has been reported by several people in the Gutenberg plugin support area. It did not exist in the 3.3 version. The error occurs (for me, anyway) on a clean install of WordPress with only the Gutenberg plugin loaded. I thought it might be related to my always turning off the visual editor in my profile, but re-enabling the option had no effect.

**Comment by pento** [comment]
@DionDesigns: This is probably caused by a security plugin or tool blocking access to `wp-tinymce.php`.

If you're using Plesk, please see #8791 for a discussion of workaround options.

**Comment by DionDesigns** [comment]
I don't think so.

This was on a clean install with the "2017" theme, and the only plugin installed was Gutenberg.. It happens on my dev VPS with no control panel, as well as on my home Windows system where I manually installed Apache, MariaDB, and PHP.

Of note is that other people have reported the same TInyMCE JS errors with this issue, and that the issue did not exist in the 3.3 version of Gutenberg.

**Comment by pento** [comment]
Can you confirm if `wp-tinymce.php` is loading, in your browser's Network debug tool?

**Comment by DionDesigns** [comment]
I just checked for wp-tinymce.php., It's trying to load wp-tinymce.php twice, and one of them contains a duplicate of the subdirectory name where WordPress is installed :

```
https://mydomain.com/wpdir/wp-includes/js/tinymce/wp-tinymce.php?c=1&ver=4800-20180716
https://mydomain.com/wpdir/wpdir/wp-includes/js/tinymce/wp-tinymce.php?ver=4.9.8
```

The second is obviously generating an HTTP 404 error. Perhaps this is an issue if WordPress is installed in a subdirectory?

**Comment by Aliyanna** [comment]
I don't think so. My WordPress is installed in the Main directory. No tiny mcs installed. Why is wp looking for it?

**Comment by Aliyanna** [comment]
Thanks. So if you deinstall Tiny MCE it shouldn't Look for it any Longer. But IT does.

**Comment by swissspidy** [comment]
No. You can't uninstall the TinyMCE editor. It's part of WordPress.

**Comment by PurTahan** [comment]
@DionDesigns can you give that to us?
i have the same problem

**Comment by tkserver** [comment]
I just barely got it to work by doing the following: 
Disable and delete the gutenberg plugin.  
Install and enable classic editor.  
Install and enable gutenberg. 
Disable classic editor.

**Comment by tkserver** [comment]
When I said it was working I may have spoken too soon.  Inserted an image via the image block, then the errors returned. 
<img width="710" alt="screen shot 2018-08-14 at 8 56 07 am" src="https://user-images.githubusercontent.com/3117151/44099711-72fe4236-9fa0-11e8-8836-79247a8b7e28.png">
<img width="826" alt="screen shot 2018-08-14 at 8 57 42 am" src="https://user-images.githubusercontent.com/3117151/44099712-731d6de6-9fa0-11e8-8c53-617f6cf5c555.png">

**Comment by Aldebaran10** [comment]
still same problem as tkserver has. I did the same steps, deleted gutenberg etc. but didn't help

**Comment by Aldebaran10** [comment]
I still have an installation with gutenberg plugin 3.6.1. With this one I don't have any problems, I can add  and work with blocks without any error messages and no js errors occur. What is the difference between 3.6.1 and 3.6.2?

**Comment by youknowriad** [comment]
I opened the #9373 PR that might fix all these issues. I'd appreciate help in testing this PR since I can't reproduce the issue myself. Thanks

**Comment by PixFX** [comment]
PROBLEM:

Suddenly I have exact the same strange blocks behaviour on existing and new posts and pages with Gutenberg Editor 3.6.2 on a clean WordPress 4.9.8 installation with no other plugins activated (as described above).

The Chrome console shows the JavaScript error "tinymce is not defined [...]", most likely because "wp-tinymce.php" is not loading, according to Chrome network tab.

(Before the problem appeared first, I had a open Chrome browser tab with a post edit screen without editing anything for around 24 hours.)

...

SOLUTION:

Logout, login and clear the browser cache solved the problem for me. 
(I used the "Cache Killer" addon for Chrome to force the clear of the browser cache.)

Hmmm, maybe some WordPress internal caching issues trigger this problem?

**Comment by youknowriad** [comment]
closed by #9373 

Feel free to raise again if it's still happening after the release of 3.7
